app/components/CrashInjuryIndicators.tsx

```diff
@@ -11,12 +13,16 @@ const InjuryBadge = ({
   className?: string;
 }) => {
   return (
-    <span className={`${className || ""} text-nowrap`}>
-      <span className="me-1">{label}</span>
-      {/* todo: decided if/how we want to assign colors to these badges */}
-      <Badge bg={`${value ? "primary" : "dark"}`} pill>
-        {value || 0}
-      </Badge>
+    <span className={`${className || ""} text-nowrap me-3`}>
+      <AlignedLabel>
+        <span className={`me-1 ${value ? "" : "text-secondary"}`}>{label}</span>
+        {value !== 0 && (
+          <Badge bg={`${value ? "dark" : "secondary"}`} pill>
+            {value}
+          </Badge>
+        )}
+        {value === 0 && <span className="text-secondary">-</span>}
```

Comment:
this stinky code is probably a hint that we should probably be using [clsx](https://github.com/lukeed/clsx)